typescript.js

```diff
@@ -1,69 +1,20 @@
-/* eslint-disable eslint-comments/disable-enable-pair */
-/* eslint-disable @typescript-eslint/naming-convention */
-// TypeScript rules with jest support
 module.exports = {
   extends: [
-    'plugin:import/typescript',
-    'plugin:@typescript-eslint/eslint-recommended',
-    'plugin:@typescript-eslint/recommended',
-    'plugin:@typescript-eslint/recommended-requiring-type-checking',
     require.resolve('./base.js'),
   ],
-  parser: '@typescript-eslint/parser',
-  plugins: ['@typescript-eslint'],
-  rules: {
-    '@typescript-eslint/naming-convention': [
-      'error',
-      {
-        selector: 'default',
-        format: ['camelCase'],
-      },
-      {
-        selector: 'enumMember',
-        format: ['PascalCase'],
-      },
-      {
-        selector: 'parameter',
-        format: ['camelCase'],
-        leadingUnderscore: 'allowSingleOrDouble',
-      },
-      {
-        selector: 'typeLike',
-        format: ['PascalCase'],
-      },
-      {
-        selector: 'variable',
-        format: ['camelCase', 'UPPER_CASE'],
-      },
-    ],
-    '@typescript-eslint/no-unused-vars': [
-      'error',
-      { argsIgnorePattern: '^_+$' },
-    ],
-    '@typescript-eslint/no-unsafe-assignment': 'warn',
-    '@typescript-eslint/no-unsafe-call': 'warn',
-    '@typescript-eslint/no-unsafe-member-access': 'warn',
-    '@typescript-eslint/restrict-template-expressions': [
-      'error',
-      { allowAny: true },
-    ],
-  },
-  overrides: [
-    {
-      /**
-       * Relax rules in test files
```

Comment:
I removed the exceptions because I think that if we agreed to use a lint rule that helps us, it should be used in all files.

One exception opens the door to enter more and more exceptions.

Reply:
The problem with removing these rules here is that test files with jest will have false positives. But I am ok trying it out again